### PR TITLE
Set of changes for Python to-do app, Cloudant version only

### PR DESCRIPTION
**Pull request by data-henrik**

### CODE REVIEW

File: python/requirements.txt [hunk 1]
```diff
@@ -14,6 +14,5 @@
 # limitations under the License.
 #
 flask==0.10.1
-pymongo==2.6.3
 apscheduler==2.1.2
-couchdb==0.9
\ No newline at end of file
+couchdb==1.1
```
Comment (l2fprod) [inline python/requirements.txt]: The Lite plan has limitations and if you go over the threshold you get error 429 from Cloudant. The nodejs cloudant library automatically retries. It seems cloudant has a package for Python doing the same auto-retry https://github.com/cloudant/python-cloudant. Should this be used here instead?

File: python/couch-app.py [hunk 2]
```diff
@@ -67,10 +67,10 @@ def index():
     return app.send_static_file('index.html')
 
 # Serve up static resources, JavaScript, HTML, CSS
-@app.route('/<path:path>')
-def js_proxy(path):
+#@app.route('/<path:path>')
```
Comment (l2fprod) [inline python/couch-app.py]: are these commented lines needed?

Comment (l2fprod) [inline python/couch-app.py]: oh now I get it, this is to serve the js/css contained in the front-end folder once I move its content here.

File: python/requirements.txt (hunk 1 above)
Comment (data-henrik) [inline python/requirements.txt]: I would keep the current library and merge the changes. We should open an issue to adapt to the Cloudant library

File: python/couch-app.py (hunk 2 above)
Comment (data-henrik) [inline python/couch-app.py]: correct - I commented those lines out, so they can be activated again once we agree on the overall structure. Until then they would be a security issue

**Comment by data-henrik** [comment]
To keep the frontend in /frontend the following needs to be changed:
1) add an extra step to recursively copy content of frontend to static
2) activate/uncomment code snippet to serve static content

**Pull request merged**